disclosure_backend/management/commands/loadalldata.py

```diff
@@ -0,0 +1,21 @@
+import glob
+import os
+from django.conf import settings
+from django.core.management import call_command
+from calaccess_raw.management.commands import CalAccessCommand
+
+
+class Command(CalAccessCommand):
+    help = 'Generate documentation for raw CAL-ACCESS database models'
```

Comment:
Either this comment is wrong, or the command is named poorly. (I would think `loadalldata` would be the combination of `downloadnetfilerawdata` and `downloadcalaccessrawdata`).